scripts/cross/bootstrap-ubuntu.sh

```diff
@@ -19,4 +21,17 @@ apt-get update
 # needed by onig_sys
 apt-get install -y \
       libclang1-9 \
-      llvm-9
+      llvm-9 \
+      unzip
+
+# Protoc. No guard because we want to override Ubuntu's old version in
```

Comment:
This block seems to be the same as `scripts/environment/install-protoc.sh`, no? Could we just call out to that?

Reply:
Yes, it is identical. I was going to open up a follow-up issue to merge these two and the copy in `soaks/Dockerfile.builder`. The biggest reason I hadn't done so already was uncertainty over which parts of `scripts` where actually available when running that bootstrap in all the places it gets run.